package.json

```diff
@@ -400,6 +400,33 @@
           "isExecutable": true,
           "description": "Specifies the full path to a PowerShell executable. Changes the installation of PowerShell used for language and debugging services."
```

Comment:
I wonder if we should modify this descript to indicate that this setting is not meant to be used by end users but is used to store the PowerShell path for the current session?